pegasus/sites.v3/code.org/public/editing/partials.md.erb

```diff
@@ -0,0 +1,485 @@
+***
+
+[breakoutquote]
+
+[col-33]
+
+<img src="/images/AR2016/best.jpg" style="max-width: 80%"/>
+
+[/col-33]
+
+[col-66]
+
+“I've never seen a nonprofit have an impact as large as this, in a timeframe as short as this. Just incredible.”
+
+*Charles Best, Founder & CEO, DonorsChoose.org*
+
+[/col-66]
+
+[clearboth]
+
+[/clearboth]
+   
+[/breakoutquote]
+
+## Side callout with colored block
+***
+
+<link href="/css/educate.css" rel="stylesheet">
+
+<div class="col-60", style="padding-right:20px;">
+
+Lorem ipsum dolor sit amet, at ius malis adversarium. In mel constituto expetendis elaboraret, tale disputationi pri no. Ne adhuc praesent nam. Mel at clita eloquentiam, sea sumo nemore facete ea, no aliquid aliquam est. Eos te virtute necessitatibus, te eos paulo audire. Pro in diam homero, rationibus cotidieque ne usu, esse dicta necessitatibus eum eu.
+
+<br>
+<br>
+His probo dolor ei, diam abhorreant ne mei. Quo ad alia eruditi, illud corpora consectetuer ut ius. Ea simul detracto nominati eum. Ex sed offendit democritum, eu ornatus fabellas nam, dolore moderatius vis ut. Nam ea quot explicari, graece viderer ponderum nam ei.
+
+</div>
+
+[col-40]
+
+<div class="educate-callout-box">
+  Looking to implement CS Fundamentals? Read this case study on how Dallas ISD brought computer science fundamentals to all schools in their district
+  <br>
+  <br>
+  <a href="/educate"><button class="educate-button" style="position: relative">Learn more</button></a>
+</div>
+
+[/col-40]
+
+<div style="clear:both"></div>
+
+<br>
+Cum liber eirmod fastidii an. Ea duo stet convenire scripserit, augue decore in sed. Vel inani utroque elaboraret ut, sit ei eius simul scaevola. Idque elitr interpretaris ut eum, numquam scaevola eu has, enim dicit per ad.
+
+Nec diceret legendos ex. An nostrud corpora perfecto vel, an sumo dictas luptatum mel. Delenit minimum ei duo, feugait qualisque deterruisset ei nam, cum ad nibh falli admodum. Brute petentium eos cu, cu mea viderer dignissim. Te fugit inimicus usu. An nec congue putant epicurei, his ad blandit detracto intellegat, natum platonem oportere te duo.
+
+Cu vis minim imperdiet. Vel cu phaedrum adolescens, vim te assentior delicatissimi, te eloquentiam voluptatibus necessitatibus est. Per te putent animal audire, liber accusam at vix. Novum nemore urbanitas eam in. Ut vim nullam aliquid temporibus, pro id unum velit omnes. Ea inermis voluptaria vix, et nonumes detraxit eam, cetero inimicus nam et.
+
+
+## Social Media
+***
+
+### How to point to Code.org's social media sites
+Do not use any social media components that are hosted on third-party sites. Third-party content can use cookies to track users, violating our privacy policy. Instead, use an image file that we host. This image may still link to the third-party site when clicked.
+
+<%= view :social_media, facebook: "Code.org", twitter: "codeorg", tumblr: "http://teacherblog.code.org", instagram: "codeorg" %>
+
+### How to share your page on social media
```

Comment:
I wonder if it's worth calling out our special centralised code for a few pages in particular?  https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/src/social_metadata.rb

Reply:
That seems worth doing! I'll leave that for a separate change, though.

These files are all available for editing via dropbox, too